spark/dl/src/main/scala/com/intel/analytics/bigdl/models/resnet/ResNet.scala

```diff
@@ -42,7 +42,7 @@ object Convolution {
      padH: Int = 0,
      nGroup: Int = 1,
      propagateBack: Boolean = true,
-     initMethod: InitializationMethod = Default,
+     initMethod: Initializer = RandomUniform,
```

Comment:
For method signature, do you mean allocating a new tensor and assign it back to the original weight instead of just setting the original weight in place. 
As for bias, filling with zero is not exactly the same as current implementation, so we need to change the reset() behavior?